ethereum/contracts/foundry.toml.in

```diff
@@ -1,10 +1,7 @@
 # See more config options https://github.com/foundry-rs/foundry/tree/master/config
 
 [profile.default]
-# Don't commit changes to this line which point to /nix/store/...
-# These are part of the `nix develop` shell setup.
-# This must be: 0.8.19
-solc = "0.8.19"
+# solc = PLACEHOLDER
```

Comment:
Does it build outside of the nix build command? e.g. open nix shell and execute `cargo build`?

Reply:
Yes, it will install solc then.